nix/haskell.nix

```diff
@@ -46,8 +46,12 @@ let
         packages.ekg.components.library.enableSeparateDataOutput = true;
         packages.cardano-node.configureFlags = [ "--ghc-option=-Werror" ];
         packages.cardano-config.configureFlags = [ "--ghc-option=-Werror" ];
-        enableLibraryProfiling = profiling;
       }
+      (lib.optionalAttrs profiling {
+        enableLibraryProfiling = true;
+        packages.cardano-node.enableExecutableProfiling = true;
+        profilingDetail = "all-functions";
```

Comment:
I think "exported-functions" should be enough. The more detail we include the slower the code runs, and the more excess detail we get in the final profiles, which can make it hard to see what is going on.

Reply:
fixed (`exported-functions` is the default)